collector/protocols/modbus.md

```diff
@@ -4,12 +4,13 @@ description: Based on MODBUS Application Protocol Specification V1.1b3
 
 # MODBUS
 
-The MODBUS protocol is capable of receiving data from a PLC (programmable logic controllers) device via TCP connections.
-
 {% hint style="info" %}
 This capability is provided by **Liverig Collector 2.28.0** or higher&#x20;
 {% endhint %}
 
+The MODBUS protocol is capable of receiving data from a PLC (programmable logic controllers) device via TCP connections if Liverig is older than 5.0.0, otherwise the data can be received  via
```

Comment:
```suggestion
The MODBUS protocol is capable of receiving data from a PLC (programmable logic controllers) device via TCP connections if LiveRig is older than 5.0.0, otherwise the data can be received  via
```